test/OptimisticArbitrator/OptimisticArbitrator.Lifecycle.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  ExpandedERC20Ethers,
+  MockOracleAncillaryEthers,
+  OptimisticOracleV2Ethers,
+  StoreEthers,
+} from "@uma/contracts-node";
+import { OptimisticArbitrator } from "../../typechain";
+import { optimisticArbitratorFixture, seedAndApprove, umaEcosystemFixture } from "../fixtures";
+import { ethers, expect, SignerWithAddress } from "../utils";
+
+let optimisticArbitrator: OptimisticArbitrator, usdc: ExpandedERC20Ethers;
+let optimisticOracle: OptimisticOracleV2Ethers, store: StoreEthers, mockOracle: MockOracleAncillaryEthers;
+let deployer: SignerWithAddress, ancillaryData: Uint8Array, liveness: number;
+
+describe("OptimisticArbitrator: Lifecycle", function () {
+  beforeEach(async function () {
+    // Load accounts and run fixtures to set up tests.
+    [deployer] = await ethers.getSigners();
+    ({ optimisticOracle, mockOracle, store } = await umaEcosystemFixture());
+    ({ optimisticArbitrator, usdc } = await optimisticArbitratorFixture());
+
+    const amountToSeedWallets = ethers.utils.parseUnits("100000", await usdc.decimals()); // 10000 USDC
+
+    liveness = 3600; // 1 hour
+    ancillaryData = ethers.utils.toUtf8Bytes(
+      `q: title: Will the price of BTC be $18000.00 or more on October 10, 2022?, description: More info. res_data: p1: 0, p2: 1, p3: 0.5, p4: -57896044618658097711785492504343953926634992332820282019728.792003956564819968. Where p1 corresponds to No, p2 to a Yes, p3 to unknown/tie, and p4 to an early request`
```

Comment:
do we need the magic number for `p4` in this example?  also do we need `p3`? it might be easier to just say `res_data 0 for no 1 for yes` to simplify this.

Reply:
also here you are asking a _question_: "will the x be y". do you think it makes more sence in this example to make this an _assertion_, like the name of this contract talks about? something like `assertion: The price of BTC was above $18000 for the duration of October 10, 2022, UTC time, considering top 5 volume weighted markets`.

curious what you think. also @smb2796 wdyt about the idealized sample assertion in this context/

Reply:
Yes makes sense to me!
p0 and p1 could be False and True, what do you think?
`assertion: The price of BTC was above $18000 for the duration of October 10, 2022, UTC time, considering top 5 volume weighted markets. res_data: p1: 0, p2: 1. Where p1 corresponds to False, p2 to True`

Reply:
yeah agreed. Also I'd say we should just remove the p1/p2 content. It's already in the YES/NO query identifier, and we'll have a separate identifier that details the fact that there are only ever 0/1 possible values.